code/__DEFINES/layers.dm

```diff
@@ -78,6 +78,19 @@
 #define MASSIVE_OBJ_LAYER 11
 #define POINT_LAYER 12
 
+
+#define EMISSIVE_BLOCKER_PLANE 12
+#define EMISSIVE_BLOCKER_LAYER 12
+#define EMISSIVE_BLOCKER_RENDER_TARGET "*EMISSIVE_BLOCKER_PLANE"
```

Comment:
All planes should probably get a render target by default

Reply:
I don't know if just setting render_target incurs any kind of performance penalty so I'll add the defines but not set them in the file without a confirmation.

Reply:
I meant that in New you should assign them a render target based on the plane number or something

Reply:
How to work `*` here